examples/fluids/meshes/cylinder.geo

```diff
@@ -35,23 +38,23 @@ DefineConstant[
 ];
```

Comment:
`resparam` is still not being used.

Reply:
I think it should be used in computing ```Layers```. But I am not sure what variable ```resparam``` should divide

Reply:
You need something like `N` as user-visible control on resolution in the plane. Once you have that, you'll have a characteristic element length, I think intended to be $h = 2H/N$, and can calculate layers as `Ceil(zspan / h)`. (I think it was confusing when you made the quadratic mesh that you made everything `N/2 + 1` so there are actually `N/2` elements along each length.)

Reply:
By "user-visible control" 
Do you mean ```N``` should be a parameter as well?

Reply:
I did it and fixed other issues like the cylinder being the wrong size because the function [is named `Sqrt`](https://gmsh.info/doc/texinfo/gmsh.html#index-Sqrt-_0028-expression-_0029) not `sqrt`. I think `cylinder.geo` is good. You should take a few minutes to open the mesh and move the sliders to see that it's sensible.